### DISCUSSION TITLE
Not accepting documentation

### DISCUSSION THREAD

**Discussion by Jade-Jones-dev** [community/community#27648] [GitHub Education]
I am a student with Code Your Future. I have tried to sign up for the student developer pack. I have used my student email as my primary email address. 

I have tried uploading the following documents and images:

Acceptance email
Student bursary
Financial aid
Screenshot of portal
Emails regarding coursework
Our syllabus
Emails from Code Your future.

None of them will pass the prechecks.

I have contacted github support who close the ticket and send me to information I have already read and tried to use to solve my issue.

Can anyone please help me.

**Comment by MrSarno** [comment]
I am sorry, however this is working as intended and we will therefore not able to assist you with this.

The [application form](https://education.github.com/discount_requests/student_application) for the GitHub Student Pack is quite clear;

<img width="672" alt="Screenshot 2022-07-31 at 11 38 55" src="https://user-images.githubusercontent.com/32073555/182022285-5a71f086-e93a-4c51-8755-e49040bfaae4.png">

Code Your Future do excellent work, however they do not issue degrees nor diplomas. They are also not an accredited institution.
